scipy/sparse/linalg/eigen/tests/test_svds.py

```diff
@@ -223,7 +223,7 @@ def test_svds_parameter_k_which(self, k, which):
         rng = np.random.default_rng(0)
         A = rng.random((10, 10))
         res = svds(A, k=k, which=which, solver=self.solver, random_state=0)
-        _check_svds(A, k, *res, which=which, atol=2e-10)
+        _check_svds(A, k, *res, which=which, atol=2e-9)
```

Comment:
The largest mismatch is `3.9e-10` it looks like, so how about we make this `5e-10` instead of `2e-9`?

Reply:
Keeping tolerances a little tight helps to detect regressions in future PRs. It's a bit of a balancing act of course, because if it's too tight then it may fails the next time someone uses a different compiler or OS. Here 5e-10 is pretty tight; 8e-10 or 1e-9 would still be reasonable. 2e-9 feels a little loose.